### DISCUSSION TITLE
insert syntax error no values in sql

### DISCUSSION THREAD

**Discussion by muyl** [actiontech/dble#2672] [Q&A] [answered]
insert into select 不支持吗

**Comment by yanhuqing666** [comment]
not a issue,but a question

**Comment by ylinzhu** [comment, marked as answer]
Support partial statement . 
For example
```
        <shardingTable name="test10" shardingNode="dn2,dn3,dn4" function="hash-three" shardingColumn="id"/>
        <shardingTable name="test11" shardingNode="dn2,dn3,dn4" function="hash-three" shardingColumn="id"/>
        <shardingTable name="test12" shardingNode="dn3,dn4" function="hash-two" shardingColumn="id"/>
        <singleTable name="test20" shardingNode="dn2" />
        <singleTable name="test22" shardingNode="dn1" />
        <globalTable name="test30" shardingNode="dn1,dn2,dn3,dn4" />
        <globalTable name="test31" shardingNode="dn1,dn2,dn3,dn4" />
```
support
  *  table of the same type with the same nodes，insert into test10(id,name) select id,name from test11; insert into test30(id,name) select id,name from test31;    
 
not supporting
  *  different types of tables， insert into test30(id,score) select id,score from test10;   
  *  tables of the same type but with different nodes， insert into test20(id,score) select id,score from test22;insert into test10(id,score) select id,score from test12;
